Examples/ScaleMicrogrid/ScaleMicrogrid.cpp

```diff
@@ -49,26 +55,31 @@ int main(int argc, char const *argv[])
  * @param Nsize - The number of DG line load cobinations to generate for scale
  * @return int returns 0 if successful, >0 otherwise
  */
-int test(index_type Nsize, bool debug_output)
+int test(index_type Nsize, real_type error_tol, bool debug_output, bool use_DAE_keys)
 {
-    real_type abstol = 1.0e-8;
-    real_type reltol = 1.0e-8;
+    real_type abstol = 1.0e-7;
+    real_type reltol = 1.0e-7;
     index_type max_step_number = 3000;
```

Comment:
This also needs to be documented. Where these numbers come from?

Reply:
Added documentation on numbers.